src/main/java/at/hannibal2/skyhanni/features/rift/area/westvillage/RiftGunthersRace.kt

```diff
@@ -105,6 +108,17 @@ object RiftGunthersRace {
         }
     }
 
+    @SubscribeEvent
+    fun onCheckRender(event: CheckRenderEntityEvent<*>) {
+        if (!(isEnabled() && config.hidePlayers)) return
```

Comment:
Shouldn't this be if !enabled || !condig.hideplayers

Reply:
https://discord.com/channels/997079228510117908/1094190239532208228/1289345953131659275
Tell me which is better to use to be consistent with the rest of the repository.